pages/common/docker-logs.md

```diff
@@ -0,0 +1,24 @@
+# docker logs
+
+> Print container logs.
+> More information: https://docs.docker.com/engine/reference/commandline/logs/.
+
+- Print logs from a container:
+
+`docker logs {{container_name}}`
+
+- Print logs and follow them:
+
+`docker logs -f {{container_name}}`
+
+- Print last {{num}} lines:
```

Comment:
I'd probably change this example to use the hard coded example, i.e.
```suggestion
- Print the last 5 lines:
```

And `--tail {{5}}` in the example.

Reply:
We definitely need to write an official style guide for things like this. :smile:

Reply:
Feel free to add it to the existing style guide. :+1:

Reply:
You're right, I missed that point. :slightly_smiling_face:

Reply:
done ✅